14/umbraco-commerce/key-concepts/events/list-of-validation-events.md

```diff
@@ -0,0 +1,469 @@
+| ValidateEmailTemplateToAddressChange | Triggered to validate changes to the TO addresses of an email template. Developers can use this event to enforce rules or validations related to TO address changes for email templates, ensuring proper configuration and management. |
+| ValidateEmailTemplateUpdate | Triggered to validate updates to an email template. Developers can use this event to enforce rules or validations related to the update process, ensuring data integrity and adherence to business logic. |
+| ValidateEmailTemplateViewChange | Triggered to validate changes to the view settings of an email template. Developers can use this event to enforce rules or validations related to view changes for email templates, ensuring proper configuration and management. |
+
+### Export Template Payment Events
+
+| **Event** | **Description** |
+|---|---|
+| ValidateExportTemplateAliasChange | Triggered to validate changes to the alias of an export template. Developers can use this event to enforce rules or validations related to the modification of export template aliases, ensuring clarity and consistency in identification. |
+| ValidateExportTemplateCategoryChange | Triggered to validate changes to the category of an export template. Developers can use this event to enforce rules or validations related to category changes for export templates, ensuring proper categorization and organization. |
+| ValidateExportTemplateCreate | Triggered to validate the creation of a new export template. Developers can use this event to enforce rules or validations related to the creation process, ensuring data integrity and adherence to business logic. |
+| ValidateExportTemplateDelete | Triggered to validate the deletion of an export template. Developers can use this event to enforce rules or validations related to the deletion process, ensuring it meets specified criteria or conditions. |
+| ValidateExportTemplateExportStrategyChange | Triggered to validate changes to the export strategy of an export template. Developers can use this event to enforce rules or validations related to export strategy changes for export templates, ensuring proper configuration and management. |
+| ValidateExportTemplateFileExtensionChange | Triggered to validate changes to the file extension of an export template. Developers can use this event to enforce rules or validations related to file extension changes for export templates, ensuring proper configuration and management. |
+| ValidateExportTemplateFileMimeTypeChange | Triggered to validate changes to the file MIME type of an export template. Developers can use this event to enforce rules or validations related to file MIME type changes for export templates, ensuring proper configuration and management. |
```

Comment:
Should we define MIME, or add it to the approve list in the styleguide?
I for one, do not know what it means 🙈

Reply:
Added MIME definition. 😀 I know MIME is globally used, but I'm not sure if the definition is widely known.